### PR TITLE
Add verifiable credential wallet

### PR DESCRIPTION
**Pull request by alexfigtree**
Hello,

Here is the Digital Credential Consortium's proposal to bring our open-source library into the OWF repository.

Please let us know if there are any questions.

### CODE REVIEW

**Review by tkuhrt — APPROVED** [review]

File: projects/verifiable-credential-wallet.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+MIT License but can change to Apache 2.0 and will leave open to discussion.
+
+# Source Control
+https://github.com/digitalcredentials/learner-credential-wallet
+
+# Issue Tracker
+https://github.com/digitalcredentials/learner-credential-wallet/issues
+
+# External Dependencies
+See package.json (https://github.com/digitalcredentials/learner-credential-wallet/blob/main/package.json) - Have ensured that all dependencies are all open source.
```
Comment (skounis) [inline projects/verifiable-credential-wallet.md:36]: Did we check for permissive and non permissive licenses?

File: projects/verifiable-credential-wallet.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+The app works with iOS and Android and allows users to add and share credentials, as well as manage the wallet.
+
+Whenver possible, we use existing open standards from W3C, IETF, and other open standards groups.
+
+
+# Alignment with the OpenWallet Foundation Mission
+This project aligns with the OWF mission, because it's an open source example implementation of wallet-related open standards.
+
+# Code of Conduct
+https://wiki.dcconsortium.org/app/page/1SYYIdVWKAdBIBx3BwSFWZKytJ9zPprAZFTbD_7Bfo4Y?p=1h1VJHHv2zSe0n9Ltg-KQevSuqWbGcExy
```
Comment (skounis) [inline projects/verifiable-credential-wallet.md:21]: Will we keep this or adopt LF's one

https://linuxfoundation.eu/en/policies

**Review by skounis — APPROVED** [review]

**Comment by tkuhrt** [comment]
@openwallet-foundation/tac : It would be great if you can review this PR. If we get a majority of people approving this from the TAC, then we can accept it; otherwise, we will need to review it at the next TAC meeting.

**Comment by tkuhrt** [comment]
@alexfigtree : We will bring this up at tomorrow's TAC meeting since we do not yet have enough votes.

**Comment by alexfigtree** [comment]
@tkuhrt Great, I'll be there with @kayelle and @dmitrizagildulin.

**Review by rolsonquadras — APPROVED** [review]

**Comment by alexfigtree** [comment]
@tkuhrt Should the project name be worked out before or after the merge?

**Comment by tkuhrt** [comment]
> @tkuhrt Should the project name be worked out before or after the merge?

Ah. Good point. We should probably figure out project name before merging this. Thanks, @alexfigtree

File: projects/verifiable-credential-wallet.md (hunk 2 above)
Comment (alexfigtree) [reply projects/verifiable-credential-wallet.md:21]: @skounis Just confirmed, we will adopt LF's.

File: projects/verifiable-credential-wallet.md (hunk 1 above)
Comment (alexfigtree) [reply projects/verifiable-credential-wallet.md]: @skounis We've checked that all dependencies are open source, which would include permissive/non permissive.

**Comment by alexfigtree** [comment]
@tkuhrt We've decided on "Learner Credential Wallet" for the name, and have updated the Code of Conduct. Please let us know if any other changes are requested.

**Comment by tkuhrt** [comment]
@alexfigtree : If you update the base branch, I can get this merged.

**Pull request merged**